### PR TITLE
Add method to check if exe is stripped

### PR DESCRIPTION
**Pull request by mpozniak95**
Connected with: https://github.com/Granulate/gprofiler/issues/804.

As far as I understand if executable has debug information it is not-stripped, and if it hasn't then it is stripped. According to pyelftools this information is stored in has_debug_info property in DWARFInfo object which can be obtained by using get_dwarf_info() method on elf file.

### CODE REVIEW

**Comment by mpozniak95** [comment]
@Jongy Please take a look

**Review by Jongy — COMMENTED** [review]
I am not sure that this is the correct property. I suggest trying to compile a dummy program with gcc, run "file" and see that it gives "not stripped". Then run "strip" on it and now you will get "stripped". Your function should behave accordingly

**Comment by mpozniak95** [comment]
Ok, you are right then. This is not correct property. I'll search for something else

File: granulate_utils/linux/elf.py [hunk 1]
```diff
@@ -183,3 +183,10 @@ def get_libc_type(elf: ELFType) -> LibcType:
             return LibcType.STATIC_LIBC
 
         return LibcType.STATIC_NO_LIBC
+
+
+def elf_is_stripped(elf: ELFType) -> bool:
```
Comment (Jongy) [inline granulate_utils/linux/elf.py:188]: This makes sense, I assume you tested it per my suggestions [here](https://github.com/Granulate/granulate-utils/pull/160#pullrequestreview-1494813829)?

File: granulate_utils/linux/elf.py [hunk 2]
```diff
@@ -183,3 +183,10 @@ def get_libc_type(elf: ELFType) -> LibcType:
...
+    with open_elf(elf) as elf:
+        if elf.get_section_by_name(".symtab") is not None:
+            return False
+        return True
```
Comment (Jongy) [inline granulate_utils/linux/elf.py:192]:
```suggestion
        return elf.get_section_by_name(".symtab") is None
```
saner

**Review by Jongy — CHANGES_REQUESTED** [review]
.

File: granulate_utils/linux/elf.py (hunk 1 above)
Comment (mpozniak95) [reply granulate_utils/linux/elf.py:188]: Yes I tested it

**Review by Jongy — APPROVED** [review]

**Pull request merged**